.github/workflows/python-sdk-schema-publish.yaml

```diff
@@ -0,0 +1,65 @@
+name: "Python: Publish @python/serverless-schema-sdk"
+
+on:
+  push:
+    tags:
+      - "@python/serverless-schema-sdk@[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  publishNewSdkVersion:
+    name: Publish new version
+    runs-on: ubuntu-latest
+    env:
+      # It'll work with secrets.GITHUB_TOKEN (which is provided by GitHub unconditionally)
+      # Still then release author would be "github-actions". It's better if it's dedicated repo bot
+      GITHUB_TOKEN: ${{ secrets.USER_GITHUB_TOKEN }}
+    steps:
+      - name: Resolve path filters
```

Comment:
What's the purpose of this step?

Reply:
I believe that's required for detecting changes to paths and so you can do things like:
```
if: steps.pathChanges.outputs.sdkSchema == 'true' || steps.pathChanges.outputs.proto == 'true'
```

Reply:
Just realized I'm looking at the publishing logic, good find 👍